pkg/kudoctl/cmd/install/install.go

```diff
@@ -28,6 +28,12 @@ func CmdErrorProcessor(cmd *cobra.Command, args []string) error {
 		return errors.WithMessage(err, "could not check kubeconfig path")
 	}
 
+	// Validate install parameters
+	err = validateInstallParameters()
+	if err != nil {
+		return errors.WithMessage(err, "could not parse parameters")
+	}
+
 	err = verifyFrameworks(args)
 	if err != nil {
```

Comment:
If we inline the error checks, we should probably also update this for consistency.

Reply:
Yes, attempted it already here as well https://github.com/kudobuilder/kudo/blob/4db20b07919ed6257a970934fe3259d25440a1b5/pkg/kudoctl/cmd/install/install.go#L32-L48